### PR TITLE
add Safer-SeqS

### PR DESCRIPTION
**Pull request by allaway**
This is currently a placeholder definition, because the SOW we reviewed said "we developed this assay..." but when I look for publications there are several different groups (none of which involve the relevant PI) that have developed a method called something like "Safer-SeqS". we should get the correct citation and add it here!

### CODE REVIEW

**Comment by jaybee84** [comment]
Yikes! I was working on a PR for this too :D

File: modules/Assay/Assay.yaml [hunk 1]
```diff
@@ -458,6 +458,8 @@ enums:
       scale:
         description: An instrument or machine for weighing.
         meaning: http://purl.obolibrary.org/obo/MMO_0000217
+      Safer-Seq:
+        description: A blood-based assay that evaluates mutations in circulating tumor DNA as well as chromosomal copy number
```
Comment (jaybee84) [inline modules/Assay/Assay.yaml:462]:
```suggestion
        description: A blood-based assay that evaluates mutations in circulating tumor DNA and chromosomal copy number changes to detect minimal residual disease (MRD) with high specificity towards tumor type.
```

File: modules/Assay/Assay.yaml [hunk 2]
```diff
@@ -458,6 +458,8 @@ enums:
       scale:
         description: An instrument or machine for weighing.
         meaning: http://purl.obolibrary.org/obo/MMO_0000217
+      Safer-Seq:
```
Comment (jaybee84) [inline modules/Assay/Assay.yaml:461]:
```suggestion
      Safer-SeqS:
```

**Review by jaybee84 — APPROVED** [review]

**Comment by allaway** [comment]
Ah, haha, sorry! See my comment on Slack - I am going to change this to SaferSeqS

**Pull request merged**